admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node can reserve a portion of its resources for use by all underlying
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
+components (e.g., *sshd*, *NetworkManager*) on a node host. Once specified, the
+scheduler has more information about the resources (e.g., memory, CPU) a node
+has allocated for pods.
+
+[[allocating-node-settings]]
+== Configuring Nodes for Allocated Resources
+
+Resources reserved for node components are based on two node settings:
+
+[options="header",cols="1,2"]
+|===
+
+|Setting |Description
+
+|`*kube-reserved*`
+| A set of `<resource_type>=<resource_quantity>` pairs (e.g.,
+`cpu=200m,memory=30G`) that describe resources reserved for node components.
+Default is none.
+
+|`*system-reserved*`
+| A set of `<resource_type>=<resource_quantity>` pairs (e.g.,
+`cpu=200m,memory=30G`) that describe resources reserved for the remaining system
+components. Default is none.
+|===
```

Comment:
IMHO a table is overkill here.

I think it can be replaced by a (factored-out) Description as intro blurb, followed by a labelled list w/ labels `kube-reserved` and `system-reserved` and text what they pertain to.

Or perhaps keep the table, but factor out the common Description text.